docs/source/guide/qse-4-low-level.md

```diff
@@ -0,0 +1,15 @@
+# What happens when I use QSE?
+A figure for the typical workflow for QSE is shown below:
+```{figure} ../img/qse-data-flow-diagram.png
+---
+width: 700px
+name: qse-workflow-overview-2
+---
+Workflow of the QSE technique in Mitiq, detailed in the [What happens when I use QSE?](qse-4-low-level.md) section.
+```
+
+The QSE workflow in Mitiq is divided into two steps: generating the code hamiltonian and the overlap hamiltonian, and then performing a classical minimization problem in order to retrieve the error-mitigated expectation value. 
```

Comment:
WDYT of something like this to break int info up a little?
```suggestion
The QSE workflow in Mitiq is divided into two steps

1. Generate the code and overlap Hamiltonian
2. Perform a classical minimization problem in order to retrieve the error-mitigated expectation value. 
```
Can you also add some details on where the minimization problem comes from?